tests/check_copyright_header.py

```diff
@@ -67,7 +67,7 @@ def main():
         if str(filename) in EXCLUSIONS:
             continue
         with open(str(filename), 'r', encoding='utf-8') as original:
-            data = original.readlines()
+            data = [line.rstrip() for line in original.readlines()]
```

Comment:
update this line on purpose because previously it did not trim the trailing `\n` symbol.

Reply:
```
ipdb> pyheader
['# Copyright (c) {CURRENT_YEAR}, NVIDIA CORPORATION.  All rights reserved.', '#', '# Licensed under the Apache License, Version 2.0 (the "License");', '# you may not use this file except in compliance with the License.', '# You may obtain a copy of the License at', '#', '#     http://www.apache.org/licenses/LICENSE-2.0', '#', '# Unless required by applicable law or agreed to in writing, software', '# distributed under the License is distributed on an "AS IS" BASIS,', '# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.', '# See the License for the specific language governing permissions and', '# limitations under the License.']
ipdb> data
 33         # If it is a comment line, we should get it
['# ! /usr/bin/python\n', '# -*- coding: utf-8 -*-\n', '# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.\n', '#\n', '# Licensed under the Apache License, Version 2.0 (the "License");\n', '# you may not use this file except in compliance with the License.\n', '# You may obtain a copy of the License at\n', '#\n', '#     http://www.apache.org/licenses/LICENSE-2.0\n', '#\n', '# Unless required by applicable law or agreed to in writing, software\n', '# distributed under the License is distributed on an "AS IS" BASIS,\n', '# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n', '# See the License for the specific language governing permissions and\n', '# limitations under the License.\n']
```